src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve(onSuccess);
+  });
+});
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(onError), 3000);
```

Comment:
You can use just reject() here because you are handling rejecting on the 30th line